src/main/java/org/broadinstitute/consent/http/service/dao/DarCollectionServiceDAO.java

```diff
@@ -37,17 +37,17 @@ public DarCollectionServiceDAO(DatasetDAO datasetDAO, ElectionDAO electionDAO, J
   }
 
   /**
-   * Find all Dar Collection + Dataset combinations that are available to the user. - Admins have
+   * Find all Dar + Dataset combinations that are available to the user. - Admins have
    * all available to them - Chairs can only create elections for datasets in their DACs
    * <p>
-   * DarCollections with no elections, or with previously canceled elections, are valid for
+   * DataAccessRequests with no elections, or with previously canceled elections, are valid for
    * initiating a new set of elections. Any DAR elections in open state should be ignored.
    *
-   * @param user       The User initiating new elections for a collection
-   * @param collection The DarCollection
+   * @param user       The User initiating new elections for a data access request
+   * @param dataAccessRequest The DataAccessRequest
    * @return List of reference ids for which a DAR election was created
    */
-  public List<String> createElectionsForDarCollection(User user, DarCollection collection)
+  public List<String> createElectionsForDarByUser(User user, DataAccessRequest dataAccessRequest)
```

Comment:
We now perform this action on a specific DAR, not the entire collection because we should not be operating on the prior data access requests that were part of the collection at this point.

Reply:
Why not call this parameter `dar`? That's what the code used before for this concept.
```suggestion
  public List<String> createElectionsForDarByUser(User user, DataAccessRequest dar)
```